source/templates/layouts/_modal.twig

```diff
@@ -0,0 +1,27 @@
+{# How to use
+  {% extends "layouts/_modal.twig" %}
+
+  {% set modal = {
+    title: 'Modal title',
+    btnSubmit: 'Add project',
+    btnReset: 'Reset'
+  } %}
+
+  {% block modalContent %}This is modal content{% endblock %}
+#}
+
+<div class="overlay"></div>
+
+<div class="modal">
+  <button class="btn-close js-btn-close" type="button">Fermer</button>
+  <h2>{{modal.title}}</h2>
+
+  <div class="modal__content">
+    {% block modalContent %}{% endblock %}
+  </div>
+
+  <div>
+    <button class="btn btn--reset" type="reset">{{modal.btnReset}}</button>
+    <button class="btn btn--add" type="submit">{{modal.btnSubmit}}</button>
```

Comment:
If we want to use "submit" and "reset" types for these buttons, they need to be inside a `<form>` tag. Of course, we don't only want 2 buttons in a form, we will also need inputs on the content of our modal, so wrap `.modal__content` and the buttons in a `<form>`